test/Cargo.toml

```diff
@@ -1,25 +1,28 @@
 [package]
 name = "jsonrpc-test"
 description = "Simple test framework for JSON-RPC."
-version = "18.0.0"
 authors = ["Tomasz Drwięga <tomasz@parity.io>"]
-license = "MIT"
-homepage = "https://github.com/paritytech/jsonrpc"
-repository = "https://github.com/paritytech/jsonrpc"
 documentation = "https://docs.rs/jsonrpc-test/"
-edition = "2018"
+version.workspace = true
+repository.workspace = true
+homepage.workspace = true
+license.workspace = true
+edition.workspace = true
 
 [dependencies]
 jsonrpc-core = { version = "18.0.0", path = "../core" }
 jsonrpc-core-client = { version = "18.0.0", path = "../core-client" }
 jsonrpc-pubsub = { version = "18.0.0", path = "../pubsub" }
-log = "0.4"
-serde = "1.0"
-serde_json = "1.0"
+log.workspace = true
+serde.workspace = true
+serde_json.workspace = true
 
 [features]
-arbitrary_precision = ["jsonrpc-core-client/arbitrary_precision", "serde_json/arbitrary_precision", "jsonrpc-core/arbitrary_precision"]
+arbitrary_precision = [
+	"jsonrpc-core-client/arbitrary_precision",
+	"serde_json/arbitrary_precision",
+	"jsonrpc-core/arbitrary_precision",
+]
 
 [dev-dependencies]
 jsonrpc-derive = { version = "18.0.0", path = "../derive" }
```

Comment:
use workspace dep?

Reply:
`jsonrpc-derive` was not in the workspace deps, but I can add it.